graphql_base/controllers/main.py

```diff
@@ -77,7 +61,7 @@ def _process_request(self, schema, data):
             return response
         except HttpQueryError as e:
             result = json_encode({"errors": [{"message": str(e)}]})
-            headers = dict(e.headers)
+            headers = dict()
```

Comment:
Hm, it looks like `e.headers` can be `None`, so I suggest this:
```suggestion
            headers = dict(e.headers or {})
```

Reply:
Done!